ci/_build_functions.sh

```diff
@@ -35,14 +35,16 @@ function push_helm_chart() {
 
   set -ex
   # this loop is to allow for concurent builds: https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1853 
-  for i in 1..${retry_count}; do
+  for i in $(seq 0 $((retry_count-1)))
+  do
     # due to helm repo index issue: https://github.com/helm/helm/issues/7363
     # we need to create new package in a different dir, merge the index and move the package back
     mkdir -p "${sync_dir}"
     helm package deploy/helm/sumologic --dependency-update --version="${version}" --app-version="${version}" --destination "${sync_dir}"
 
     git fetch "${remote}" gh-pages
     git stash push
+    git pull "${remote}" gh-pages
```

Comment:
Why does this help? We're already fetching it two lines above, and then checking it out the line below.

Reply:
I'm going by the build logs for this change

```
o https://github.com/SumoLogic/sumologic-kubernetes-collection
 ! [rejected]          gh-pages -> gh-pages (fetch first)
error: failed to push some refs to 'https://github.com/SumoLogic/sumologic-kubernetes-collection'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
Pruning the dev Helm Chart releases older than one month
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
```

Reply:
I removed it